### PR TITLE
Add toggle for prompts in docstring examples next to copybutton

### PR DESCRIPTION
**Pull request by lagru**
## Description

As discussed in the [last meeting](https://github.com/scikit-image/meeting-notes/blob/master/2019/2019-10-28.markdown) sphinx_copybutton has the disadvantage that prefixes such as `>>> ` and output are copied as well. 

I propose to replace this with a simple javascript copied from [python/python-docs-theme](https://github.com/python/python-docs-theme/blob/b862f1159388ed3134818e56b065e18b835d652c/python_docs_theme/static/copybutton.js). The replacement allows to hide these and allows the user to select content by himself. This should make copying and running examples inside your own console a bit less tedious.

Not sure if I accommodated the license here in the right way. Do we need to include tools that are not included in the distribution archives in the license as well (e.g. as with tifffile)?

E.g. [skimage.exposure.histogram](https://scikit-image.org/docs/stable/api/skimage.exposure.html#histogram):

image
after clicking the button
image

## Checklist

- [x] [Docstrings for all functions](https://github.com/numpy/numpy/blob/master/doc/example.py)
- [x] Clean style in [the spirit of PEP8](https://www.python.org/dev/peps/pep-0008/)

## For reviewers

- [ ] Check that the PR title is short, concise, and will make sense 1 year
  later.
- [ ] Check that new functions are imported in corresponding `__init__.py`.
- [ ] Check that new features, API changes, and deprecations are mentioned in
      `doc/release/release_dev.rst`.
- [ ] Consider backporting the PR with `@meeseeksdev backport to v0.14.x`

cc @jni, @alexdesiqueira

### CODE REVIEW

**Comment by hmaarrfk** [comment]
I think you need to include the PSF license in the JS file somewhere maybe?

**Comment by lagru** [comment]
@hmaarrfk Good idea, can do!

**Comment by hmaarrfk** [comment]
This file is only used to a generate an other file correct? Therefore the license won't be shipped to every user on every page correct?

**Comment by lagru** [comment]
It's included in every generated HTML page with `<script type="text/javascript" src="_static/copybutton.js"></script>`. That means it's loaded on every page. :/ Not sure what to do about that. `app.add_js_file` only allows to include a script file globally.

**Comment by hmaarrfk** [comment]
Whatever, let's let the content delivery network guys worry about shipping the license text to the user.

**Comment by lagru** [comment]
Sorry? :sweat_smile: What do you mean by that? 

[Test failure](https://travis-ci.org/scikit-image/scikit-image/jobs/604497522#L2563) looks unrelated.

**Comment by hmaarrfk** [comment]
Typically people try to avoid the extra kilobyte of data from the license text. But that is a problem for the guys hosting the servers. Not us.

**Comment by jni** [comment]
@lagru is there no way to have both buttons together? Maybe @choldgraf has ideas/opinions here. ;)

Other than that, looks good!

**Comment by lagru** [comment]
@jni I think I figured it out:

image
image

I tried to get both buttons as consistent as possible but I'm a real beginner with JS and CSS. I couldn't get the tooltip to align in the same way that it does for the copybutton.

**Comment by lagru** [comment]
@hmaarrfk Concerning the license: I opted to clearly state the license and a link to it in the scripts header. This seems to be standard practice when looking at other scripts loaded on our page.

File: doc/source/themes/scikit-image/static/css/custom.css [hunk 1]
```diff
@@ -297,6 +297,30 @@ p.sphx-glr-signature a.reference.external {
...
 }
 
+
+span.togglebutton {
+    cursor: pointer;
+    position: absolute;
+    top: 6px;
+    right: 25px;
+    font-family: "monospace";
+    line-height: 100%;
```
Comment (lagru) [inline doc/source/themes/scikit-image/static/css/custom.css]: I'm not sure whether the absolute positioning, padding, line-height and such will translate gracefully to other browsers and machines. Any CSS experts that have more insight?

**Comment by hmaarrfk** [comment]
>  This seems to be standard practice when looking at other scripts loaded on our page.

It can be standard practice, but it doesn't mean that it is exactly correct.

According to point #2:
> provided, however, that PSF's License Agreement and PSF's notice of copyright

So you've included the notice of copyright, but not the PSF License Agreement itself.

I imagine much of the internet is tainted by this, people using part of other's code all the time without following 100% of the license agreement.

**Comment by alexdesiqueira** [comment]
Thanlks @lagru! That'll be a great addition to our site :)

**Comment by jni** [comment]
@meeseeksdev backport to v0.16.x

**Pull request merged**